src/lib/consensus/proof_of_stake.ml

```diff
@@ -8,7 +8,7 @@ open Signature_lib
 open Snark_params
 open Num_util
 module Time = Block_time
-module Run = Snark_params.Tick.Run
+(* module Run = Snark_params.Tick.Run *)
```

Comment:
General remark: I understand the intention of commenting instead of removing unused modules, but IMO they should be removed when applicable. Such comments clutter the code base and end up being confusing and the modules can always be retrieved with source control.